include/wx/string.h

```diff
@@ -75,6 +75,10 @@
 
 class WXDLLIMPEXP_FWD_BASE wxString;
 
+#ifdef wxUSE_REGEX
```

Comment:
This looks like a leftover from the old version, AFAICS all the changes to this header should be reverted completely now.